xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/suggest/xwiki.selectize.js

```diff
@@ -205,6 +205,11 @@ define('xwiki-selectize', ['jquery', 'selectize', 'xwiki-events-bridge'], functi
     if (this.selectize.settings.takeInputWidth) {
       this.selectize['$control'].width($(this).data('initialWidth'));
     }
+    // Set the title of the input field if it does exists in the select.
+    let inputTitle;
```

Comment:
Note that right now are functional test are executed on an old version of Firefox, which does not support the "let" keyword, see the comment from @mflorea: https://jira.xwiki.org/browse/XCOMMONS-862?focusedCommentId=98996&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-98996

Now we recently upgraded Guava which mean we should be able to upgrade Selenium and our version of firefox to run those functional test... So I guess we need to agree on that before merging this code.

Reply:
Then I will wait to see what's proposed for the upgrade. If it's not planned any time soon, I can change it to 'var'.

Reply:
Yes, better use ``var`` for now, but I'm not sure we need a variable. Do we really need the ``if``? Why not copy directly the value of the ``title`` attribute?

Reply:
Indeed. This way I also won't need let/ var